### PR TITLE
Persona page updates

### PR DESCRIPTION
**Pull request by ethangj**

### CODE REVIEW

File: collections/developers.html [hunk 1]
```diff
@@ -64,6 +79,15 @@ <h2 class="box__title">Using The API</h2>
         </div>
       </div>
 
+      <div class="homepage-header">
+        <h3>Case Study</h3>
+      </div>
+      <div id="case-study">
+        <p>Several companies have published case studies on using Codeship to improve their operations.</p>
+        <p><a href="
+          https://codeship.com/case-studies/placester?utm_source=TravelPerkCaseStudy">Read about Placester</a> and then read the others to learn more about what Codeship can do for you!</p>
```
Comment (joesiewert) [inline collections/developers.html]: Link is for `placester` or `travelperk`?

File: collections/devops.html [hunk 2]
```diff
@@ -87,6 +154,38 @@ <h2 class="box__title">Support</h2>
         </div>
       </div>
 
+      <div class="homepage-header">
+        <h3>Case Study</h3>
+      </div>
+      <div id="case-study">      
+        <p>Several companies have published case studies on using Codeship to improve their operations.</p>
+        <p><a href="
+          https://codeship.com/case-studies/placester?utm_source=TravelPerkCaseStudy">Read about Placester</a> and then read the others to learn more about what Codeship can do for you!</p>
```
Comment (joesiewert) [inline collections/devops.html]: Link mismatch again.

File: collections/manage-teams.html [hunk 3]
```diff
@@ -62,6 +76,15 @@ <h2 class="box__title">Integrating Tools And Services</h2>
         </div>
       </div>
 
+      <div class="homepage-header">
+        <h3>Case Study</h3>
+      </div>
+      <div id="case-study">
+        <p>Several companies have published case studies on using Codeship to improve their operations.</p>
+        <p><a href="
+          https://codeship.com/case-studies/placester?utm_source=TravelPerkCaseStudy">Read about Placester</a> and then read the others to learn more about what Codeship can do for you!</p>
```
Comment (joesiewert) [inline collections/manage-teams.html]: Link mismatch.

**Comment by ethangj** [comment]
@joesiewert Good catch, fixed

**Pull request merged**